### PR TITLE
♻️ Adds comment to toolbarElements

### LINKED ISSUE #18016: Demo Issue for Contributor Submit
Could we add a comment here to explain what is the `toolbarElements`?

https://github.com/ampproject/amphtml/blob/d3df64d07ae9e9e21a94d8339f3c2d0a9b7d4261/extensions/amp-sidebar/0.1/amp-sidebar.js#L126

### PR DESCRIPTION
**Pull request by ampcontributorsummit2424**
closes #18016 

This adds a comment on the addition of toolbars to sidebar, for clearer code.

### CODE REVIEW

**Comment by ampcontributorsummit2424** [comment]
I signed it!

**Comment by ampcontributorsummit2424** [comment]
Hello, @torch2424 I have completed the issue! 😄

File: extensions/amp-sidebar/0.1/amp-sidebar.js [hunk 1]
```diff
@@ -123,6 +123,8 @@ export class AmpSidebar extends AMP.BaseElement {
     // Get the toolbar attribute from the child navs.
     const toolbarElements = toArray(element.querySelectorAll('nav[toolbar]'));
 
+    // Add found toolbar elements, as new Toolbars.
```
Comment (torch2424) [inline extensions/amp-sidebar/0.1/amp-sidebar.js]: nit: Instead of a period, could we do a comma?

Comment (ampcontributorsummit2424) [reply extensions/amp-sidebar/0.1/amp-sidebar.js]: Good point. Could we do a semi colon?

Comment (torch2424) [reply extensions/amp-sidebar/0.1/amp-sidebar.js]: Semi Colons are cool. But I think period would be best here.

**Comment by ampcontributorsummit2424** [comment]
This is now ready for review again 😄

File: extensions/amp-sidebar/0.1/amp-sidebar.js (hunk 1 above)
Comment (ampcontributorsummit2424) [reply extensions/amp-sidebar/0.1/amp-sidebar.js]: Done.

**Comment by ampcontributorsummit2424** [comment]
Closing this as it was simply made for a talk 😄

**Pull request closed without merging**